TODO

```diff
@@ -27,8 +27,6 @@ API additions
 
 - Add query / enumeration API for aalt-like features?
 
-- SFNT api? get_num_faces?
```

Comment:
Do we another things like `ot.get_tag` maybe? That particular one is just a detail I guess

Reply:
No. We're good now that there's get_table_tags and this.